src/sympc/grads/grad_functions.py

```diff
@@ -636,6 +636,7 @@ def forward(
         stride: Optional[Union[int, Tuple[int, int]]] = None,
         padding: Union[int, Tuple[int, int]] = 0,
         dilation: Union[int, Tuple[int, int]] = 1,
+        return_indices: bool = False,
```

Comment:
Missed out forwarding this arg to the call below (L679)

https://github.com/OpenMined/SyMPC/blob/969718db81690b674a642b29984074fd45286034/src/sympc/grads/grad_functions.py#L673-L680

Reply:
I should have had an if...else statement - the return indices would be needed if we record the gradient.